<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MTFileTranslate</class>
 <widget class="QWidget" name="MTFileTranslate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MTFileTranslate</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>529</width>
     <height>99</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="lbIPC">
        <property name="text">
         <string>IP:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leIPC">
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbPC">
        <property name="text">
         <string>端口:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lePC">
        <property name="text">
         <string>9999</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStartClient">
        <property name="text">
         <string>连接服务</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="lbPathC">
        <property name="text">
         <string>路径:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lePathC"/>
      </item>
      <item>
       <widget class="QCheckBox" name="cbC">
        <property name="text">
         <string>文件夹</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSelectC">
        <property name="text">
         <string>选择</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="lbJDC">
        <property name="text">
         <string>进度:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leJDC"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>0</y>
     <width>529</width>
     <height>99</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lbIPS">
        <property name="text">
         <string>IP:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leIPS">
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbPS">
        <property name="text">
         <string>端口:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lePS">
        <property name="text">
         <string>9999</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStartServer">
        <property name="text">
         <string>启动服务</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lbPathS">
        <property name="text">
         <string>路径:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lePathS"/>
      </item>
      <item>
       <widget class="QCheckBox" name="cbS">
        <property name="text">
         <string>文件夹</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSelectS">
        <property name="text">
         <string>选择</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lbJDS">
        <property name="text">
         <string>进度:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leJDS"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
